docs/src/DeveloperDocumentation/styleguide.md

```diff
@@ -184,6 +184,53 @@ end
 
 However, as always, rules sometimes should be broken.
 
+## Optional arguments for parents of return values
+
+Several objects in Oscar have `parent`s, e.g. polynomials, group elements, ... 
```

Comment:
Please use `OSCAR` instead of `Oscar` whenever talking about the computer algebra system (while `Oscar` is reserved for the Julia package itself; but this is a subtle distinction, if in doubt, write `OSCAR`).

(Again, not a blocker, but I hope changing it won't be hard)